e2e/commands/add-many.e2e.js

```diff
@@ -118,4 +124,55 @@ describe('bit add many programmatically', function () {
       expect(nodeStartOutputObj).to.be.ofSize(5);
     });
   });
+  describe('make sure gitignore is read from the correct path', function () {
+    let newDirPath;
+    before(function () {
+      helper.reInitLocalScope();
+      helper.copyFixtureComponents('add-many');
+      helper.createFile('foo', 'c.js');
+      helper.createFileOnRootLevel('c.js');
+      helper.npmLink('bit-bin');
+      newDirPath = helper.createNewDirectory();
```

Comment:
make sure new dir is deleted in the end (and not deleted if run with --keep-envs)